obm-autoset/obm_config.json

```diff
@@ -0,0 +1,23 @@
+{ "ora": {

```

Comment:
Please fix the indent of this JSON file, so as to make it more friendly to user.